README.md

```diff
@@ -48,4 +48,34 @@ It will produce the following output:
   ####   #    #   #####  #        ####   #    #     ##  
                                                         
                                                         
+```
+
+### Usage with Arabic
+_NOTE: This library handles Arabic scripts, but you need to use the Arabic processors as shown in this example._
+
+Example code is under `examples/arabic.rs`.
+Run the binary with the following command:
+```
+cargo run --example arabic 'أبجد ABC ١٢٣ لا 123'
```

Comment:
[NOTE] The example text is not rendered correctly since Github doesn't seem to handle bidirectional text correctly. 
The correct display should be 

<img width="227" alt="Screen Shot 2024-03-31 at 10 33 49" src="https://github.com/mkovaxx/unifont-rs/assets/25980652/851e1d11-e902-4126-9677-eb63de6c4e7d">